src/components/group-common.js

```diff
@@ -38,37 +38,48 @@ export const AdminNotifierCheckbox = ({ checked, onChange }) => {
 }
 
 export const NewUserCard = ({ onClick }) => {
-  return h(Clickable, {
-    style: Style.cardList.shortCreateCard,
+  return h(ButtonPrimary, {
+    style: { textTransform: 'none' },
     onClick
   }, [
-    div(['Add a User']),
-    icon('plus-circle', { style: { marginTop: '0.5rem' }, size: 21 })
+    icon('plus', { size: 14 }),
+    div({ style: { marginLeft: '0.5rem' } }, ['Add User'])
+  ])
+}
+
+const UserMenuContent = ({ onEdit, onDelete }) => {
+  return h(Fragment, [
+    h(MenuButton, {
+      onClick: () => onEdit()
+    }, [makeMenuIcon('edit'), 'Edit Role']),
+    h(MenuButton, {
+      onClick: () => onDelete()
+    }, [makeMenuIcon('trash'), 'Remove User'])
   ])
 }
 
 export const MemberCard = Utils.memoWithName('MemberCard', ({ member: { email, roles }, adminCanEdit, onEdit, onDelete, adminLabel, userLabel }) => {
   const canEdit = adminCanEdit || !_.includes(adminLabel, roles)
   const tooltip = !canEdit && `This user is the only ${adminLabel}`
+  const menuSize = 20
 
   return div({
-    style: Style.cardList.longCard
+    style: { ...Style.cardList.longCard, boxShadow: 'none' }
```

Comment:
I wanted to explicitly have `boxShadow: 'none'` everywhere, instead of abstracting, because even if it is in every place that longCard is used, it breaks our general card patterns, and I didn't want someone to use `longCard` expecting standard card patterns and get otherwise.

Reply:
Could we have a different card name or something? Like `longCardShadowless`? It seems like in the interest of DRY, we should abstract it if we have it in every place that we use `longCard`, but I also agree with what you're saying about having `longCard` conform with our general card patterns

Reply:
Yeah, I can do that.